src/theme.ts

```diff
@@ -289,3 +289,18 @@ export const newTheme = extendTheme({
     },
   },
 });
+
+console.log(newTheme.components);
+
+extendTheme({
+  components: {
+    MuiCard: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          backgroundColor: White,
+          [theme.getColorSchemeSelector('dark')]: { backgroundColor: TestTool },
```

Comment:
I think within the theme, you want to use the MUI theme colors, not the external colors. For example: 
```suggestion
          backgroundColor: theme.vars.palette.neutral.light,
```

Reply:
Cited from the [MUI docs](https://mui.com/material-ui/experimental-api/css-theme-variables/customization/#components)

Reply:
This is outdated, I ended up adding the following lines to the dark theme, which overrides the colors used by cards and some other MUI text components:
```    
dark: {
      // Palette for dark theme mode -------------------------------------------------------
      palette: {
        background: {
          paper: TestTool, // Card Colors
        },
        text: {
          primary: TestToolContrast, // Various MUI components and text
        },
```